core/src/main/java/io/questdb/cutlass/http/processors/SettingsProcessor.java

```diff
@@ -54,7 +54,7 @@ public class SettingsProcessor implements HttpRequestHandler {
     // processor. For different threads to lookup the same value from local value map the key,
     // which is LV, has to be the same between processor instances
     private static final LocalValue<SettingsProcessorState> LV = new LocalValue<>();
-    private static final LocalValue<StringSink> LV_GET = new LocalValue<>();
+    private static final LocalValue<Utf8StringSink> LV_GET = new LocalValue<>();
```

Comment:
it would be good to name these local values after what they hold, LV_GET is a bit random

Reply:
done in https://github.com/questdb/questdb/pull/5701/commits/ab17ea60ba7428931122eb61b2fabb83ff280917